sw/device/silicon_creator/lib/drivers/pinmux_unittest.cc

```diff
@@ -71,6 +73,13 @@ class InitTest : public PinmuxTest {
   };
 };
 
+TEST_F(InitTest, PadAttrPropagationDelay) {
+  const uint64_t kCpuClockPeriodNs =
+      (1.0 / static_cast<double>(kClockFreqCpuHz)) * 1000000000;
```

Comment:
nit: you don't really need the cast to double and the division if the result will be stored in an integer - please note the digit separator `'` which was added in c++14:
```suggestion
  const uint64_t kCpuClockPeriodNs = 1'000'000'000 / kClockFreqCpuHz;
```

Reply:
Big +1 to eliminating floating point whenever possible. I've been burned so many times!

Reply:
fixed.